socs/agents/galildmc_axis_controller/drivers.py

```diff
@@ -0,0 +1,504 @@
+            if axis not in axes:
+                continue  # skip axes user doesn't care about
+
+            data[axis] = {
+                'position': tp[i],
+                'velocity': tv[i],
+                'torque': tt[i],
+                'position_error': te[i]
+            }
+
+        return data
+
+    def is_running(self, axis):
+        """
+        Checks if the axis is moving.
+
+        """
+        cmd = f'MG _BG{axis}'
+        resp = self.galil_command(cmd, expect_response=True)
+        return resp
+
+    def begin_motion(self, axis):
+        """
+        Begin motion for the specified axis using the BG command.
+
+        """
+        self.galil_command(command="BG", axis=axis)
+
+        state = self.is_running(axis)
+        state = str(int(float(state)))
+        if state == '1':
+            print(f'Axis {axis} is in motion.')
+        elif state == '0':
+            print(f'Axis {axis} is not moving. Retry command if needed.')
+
+    def set_relative_position(self, axis, distance, counts_per_unit=None):
+        """
+        Move axis by a relative distance (in calibration units such as
+        mm or deg or raw encoder counts). If `counts_per_unit` is None,
+        assume raw counts.
+
+        """
+        counts = distance if counts_per_unit is None else round(distance * counts_per_unit, 3)
+        return self.galil_command("PR", axis=axis, value=counts)
+
+    def set_absolute_position(self, axis, position, counts_per_unit=None):
+        """
+        Set absolute position for an axis in units or encoder counts.
+
+        """
+        counts = position if counts_per_unit is None else round(position * counts_per_unit, 3)
+        return self.galil_command("PA", axis=axis, value=counts)
+
+    def release_brake(self, output_num):
+        """
+        Release brake to axis by using the GalilDMC SB command which sets the digital
+        output to 1.
+
+        """
+        resp = self.galil_command(command="SB", value=output_num)
+        return resp
+
+    def engage_brake(self, output_num):
+        """
+        Engage the brake for the specified axis using the Galil CB command.
+
+        """
+        resp = self.galil_command(command="CB", value=output_num)
+        return resp
+
+    def get_brake_status(self, axis, output_map):
+        """
+        Return brake status for axis via @OUT[n].
+
+        """
+        output_num = output_map[axis]
+        query_str = f"@OUT[{output_num}]"
+        val = self.galil_command(command="MG", value=query_str)
+
+        try:
+            state = int(round(float(val)))
+        except (TypeError, ValueError):
+            print(f"Could not parse brake value '{val}' for axis {axis}.")
+            return {axis: {"state": None, "status": "Unknown"}}
+
+        if state == 1:
+            status = "Brake Released"
+        if state == 0:
+            status = "Brake Engaged"
+
+        return {axis: {"state": state, "status": status}}
```

Comment:
Since this is always called for a single axis, just return `state` and `status` (you could also just return `status` only)
```suggestion
        return state, status
```